SA3/qa_web/templates/qa_web/question_thread.html

```diff
@@ -24,52 +24,51 @@ <h1 class="masthead-heading mb-0">
                 </span>
             </h1>
             <h5 class="masthead-subheading mb-0">
-                <p><span class="marked" id="content_question">{{ currentQuestion.content }}</span></p>
+                <p><span class="marked" id="content_question">
+                    {{ currentQuestion.content }}
+                </span></p>
```

Comment:
Here, the span tag should be all on one line as it was before. The spaces and line breaks within the `span` tag makes our code vulnerable to the bug we had before: Scrollbars appearing on the content if MD processed it as `<code>`.